locale/templates/settings.ftl

```diff
@@ -0,0 +1,399 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Globally reused strings
+
+# Note: the company name, Mozilla, is not translated.
+-mozilla = Mozilla
+# Note: the product name, Firefox Accounts, is not translated.
+-firefox-accounts = Firefox Accounts
+-firefox-account = Firefox account
+
+# Frequently reused button text. The translations should be short enough to fit in a small button.
+fxa-save-button = Save
+fxa-close-button = Close
+fxa-cancel-button = Cancel
+fxa-continue-button = Continue
+
+# Application
+
+app-default-title = { -firefox-accounts }
+app-page-title = { $title } | { -firefox-account }
+
+# Application-wide footer
+
+app-footer-mozilla-logo-label = { -mozilla } logo
+app-footer-privacy-notice = Website Privacy Notice
+app-footer-terms-of-service = Terms of Service
+
+# Avatar component
+
+avatar-your-avatar =
+  .alt = Your avatar
+avatar-default-avatar =
+  .alt = Default avatar
+
+## Connect another device promo
+
+connect-another-fx-mobile = Get Firefox on mobile or tablet
+connect-another-find-fx-mobile = Find Firefox in the Google Play and App Store
+  or <br /><linkExternal>send a download link to your device.</linkExternal>
+
+## Connected services section
+
+cs-heading = Connected Services
+cs-description = Everything you are using and signed into.
+cs-cannot-refresh = Sorry, there was a problem refreshing the list of connected
+  services.
+cs-cannot-disconnect = Client not found, unable to disconnect
+cs-logged-out = Logged out of { $service }.
```

Comment:
There needs to be a reference of the variable at the beginning of this block, as seen in the example here: https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#comments